src/components/MapCardList/index.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
```

Comment:
I think we should still name the files the same name as the folder just so it's obvious when looking at the filename, unless that's bad practice?